alf/bin/play.py

```diff
@@ -60,10 +60,11 @@
 def main(_):
     gin_file = common.get_gin_file()
     gin.parse_config_files_and_bindings(gin_file, FLAGS.gin_param)
-    algorithm_ctor = gin.query_parameter(
-        'TrainerConfig.algorithm_ctor').scoped_configurable_fn
+    algorithm_cls = gin.query_parameter(
```

Comment:
Does "_cls" here mean "class"? Maybe change to something like "algorithm_type" is more easier for understanding.

Reply:
Personly think cls is better as we configured a class not  str type here

Reply:
the original algorithm_ctor might be better, because it's still conceivable that we use a function to create the algorithm in the future.

Reply:
i have renamed it back to `algorithm_ctor`